app/src/main/kotlin/com/wire/android/ui/home/conversations/model/MessageViewWrapper.kt

```diff
@@ -17,7 +17,7 @@ enum class MessageStatus(val stringResourceId: Int) {
     Untouched(-1), Deleted(R.string.label_message_status_deleted), Edited(R.string.label_message_status_edited)
 }
 
-data class Message(
+data class MessageViewWrapper(
```

Comment:
confusing name to me a bit, what does it wrap, why would it wrap anything ?

Reply:
I added wrapper since it also contains info like `messageSource` that is not directly related with the UI. I can also rename to `messageUI` if preferred